src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -208,6 +208,9 @@ const UniqueTokenExpandedState = ({
   } = useShowcaseTokens();
 
   const [floorPrice, setFloorPrice] = useState<string | null>(null);
+  const [priceOfEth, setPriceOfEth] = useState<number>(
+    ethereumUtils.getEthPriceUnit()
+  );
```

Comment:
I guess it should be something like this so we don't run this thing every rerender.

```suggestion
  const [priceOfEth, setPriceOfEth] = useState<number>(
    () => ethereumUtils.getEthPriceUnit()
  );
```

Reply:
🙇